Unable to build frontend.

<!-- NOTE: If your issue is a security concern, please send an email to security@gitea.io instead of opening a public issue -->

<!--
    1. Please speak English, this is the language all maintainers can speak and write.
    2. Please ask questions or configuration/deploy problems on our Discord
       server (https://discord.gg/gitea) or forum (https://discourse.gitea.io).
    3. Please take a moment to check that your issue doesn't already exist.
    4. Make sure it's not mentioned in the FAQ (https://docs.gitea.io/en-us/faq)
    5. Please give all relevant information below for bug reports, because
       incomplete details will be handled as an invalid report.
-->

- Gitea version (or commit ref): 25b6255b924c30a2bab066c0465d146a34021245
- Git version: 2.30.0
- Operating system: Manjaro Linux
  <!-- Please include information on whether you built gitea yourself, used one of our downloads or are using some other package -->


  <!-- Please also tell us how you are running gitea, e.g. if it is being run from docker, a command-line, systemd etc. --->
  <!-- If you are using a package or systemd tell us what distribution you are using -->
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
- Log gist:
<!-- It really is important to provide pertinent logs -->
<!-- Please read https://docs.gitea.io/en-us/logging-configuration/#debugging-problems -->
<!-- In addition, if your problem relates to git commands set `RUN_MODE=dev` at the top of app.ini -->

## Description

The following command (as is listed in the README) results in the following error, this is after running pacman -Syu.

```
[allie@allie-virtualbox gitea]$ TAGS="bindata sqlite sqlite_unlock_notify" make build
rm -rf public/js public/css public/fonts public/img/webpack public/serviceworker.js
npx webpack
/home/allie/Gitea/gitea/node_modules/webpack/node_modules/source-map/lib/util.js:24
    throw new Error('"' + aName + '" is a required argument.');
          ^

Error: "version" is a required argument.
    at Object.getArg (/home/allie/Gitea/gitea/node_modules/webpack/node_modules/source-map/lib/util.js:24:11)
    at new BasicSourceMapConsumer (/home/allie/Gitea/gitea/node_modules/webpack/node_modules/source-map/lib/source-map-consumer.js:294:22)
    at new SourceMapConsumer (/home/allie/Gitea/gitea/node_modules/webpack/node_modules/source-map/lib/source-map-consumer.js:22:7)
    at SourceMapSource.node (/home/allie/Gitea/gitea/node_modules/webpack/node_modules/webpack-sources/lib/SourceMapSource.js:193:4)
    at exports.getSourceAndMap (/home/allie/Gitea/gitea/node_modules/webpack/node_modules/webpack-sources/lib/helpers.js:20:27)
    at SourceMapSource.sourceAndMap (/home/allie/Gitea/gitea/node_modules/webpack/node_modules/webpack-sources/lib/SourceMapSource.js:184:10)
    at getTaskForFile (/home/allie/Gitea/gitea/node_modules/webpack/lib/SourceMapDevToolPlugin.js:78:30)
    at /home/allie/Gitea/gitea/node_modules/webpack/lib/SourceMapDevToolPlugin.js:266:22
    at /home/allie/Gitea/gitea/node_modules/webpack/lib/Cache.js:93:5
    at Hook.eval [as callAsync] (eval at create (/home/allie/Gitea/gitea/node_modules/tapable/lib/HookCodeFactory.js:1:1), <anonymous>:6:1)
    at Cache.get (/home/allie/Gitea/gitea/node_modules/webpack/lib/Cache.js:75:18)
    at ItemCacheFacade.get (/home/allie/Gitea/gitea/node_modules/webpack/lib/CacheFacade.js:116:15)
    at /home/allie/Gitea/gitea/node_modules/webpack/lib/SourceMapDevToolPlugin.js:217:18
    at arrayEach (/home/allie/Gitea/gitea/node_modules/neo-async/async.js:2405:9)
    at Object.each (/home/allie/Gitea/gitea/node_modules/neo-async/async.js:2846:9)
    at /home/allie/Gitea/gitea/node_modules/webpack/lib/SourceMapDevToolPlugin.js:204:15
make: *** [Makefile:663: public/js/index.js] Error 1
```

...


## Screenshots

<!-- **If this issue involves the Web Interface, please include a screenshot** -->

--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
     "codemirror": "5.59.0",
     "core-js": "3.8.1",
     "css-loader": "5.0.1",
-    "css-minimizer-webpack-plugin": "1.1.5",
+    "css-minimizer-webpack-plugin": "1.2.0",
     "dropzone": "5.7.2",
     "easymde": "2.13.0",
     "escape-goat": "3.0.0",

